### DISCUSSION TITLE
With superset installed from git clone unable to find the site-packages?

### DISCUSSION THREAD

**Discussion by aakanshajecrc** [apache/superset#18927] [Q&A / Help]
I have superset installed from a git clone.

All seems to be working ok but I don't have the site-packages. For example File "/usr/local/lib/python3.8/site-packages/flask/cli.py" is missing

I have been looking around the official documentation but I cannot  find how.

**Comment by srinify** [comment]
Hi @aakanshajecrc we're reserving Github issues for discussion around bugs so I'm moving this over to Discussions!
